kubernetes/windows/Dockerfile

```diff
@@ -79,5 +80,7 @@ ENV APPLICATIONINSIGHTS_AUTH "NzAwZGM5OGYtYTdhZC00NThkLWI5NWMtMjA3ZjM3NmM3YmRi"
 ENV AZMON_COLLECT_ENV False
 ENV CI_CERT_LOCATION "C://oms.crt"
 ENV CI_KEY_LOCATION "C://oms.key"
+ENV MONITORING_DATA_DIRECTORY "C:\\opt\\genevamonitoringagent\\datadirectory"
+ENV MONITORING_MCS_MODE "1"

```

Comment:
We will running AMA in 1P & 3P mode, this should be moved to main.ps1 so that this can be set accordingly based on the configured destination mode.